challenges/objects-arrays.js

```diff
@@ -89,32 +113,42 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
-console.log(animalNames);
+let names = zooAnimals.forEach(function(i){
+  zooAnimals.push(i.scientific_name + i.animal_name);
+});
+console.log(zooAnimals);
 
 /* Request 2: .map()    
 
 The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
 */
-
-const lowerCase = [];
+const lowerCase = animalNames.map((zooAnimal) => {
+  return scientific_name.toLowerCase();
+});
 console.log(lowerCase); 
 
+
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const largerPopulation = [];
-console.log(largerPopulation);
+// const largerPopulation = zooAnimals.filter((pop) => {
+//   return pop.zooAnimals === 5;
```

Comment:
Do each of your objects in the zooAnimals array have a property called zooAnimals? Also, right now you are checking if that is equal to five, while it asks for the animals with a population less than five.